core/shared/src/test/scala/tests/DateFormatSpec.scala

```diff
@@ -74,11 +76,11 @@ class DateFormatSpec extends WordSpec with Matchers {
     }
 
     "retrieve miscellaneous info" in {
-      gmtDate.t.timeZoneOffsetMM // can only be tested by directly redefining the method
-      gmtDate.t.timeZoneOffsetHH // can only be tested by directly redefining the method
-      gmtDate.t.timeZone // relative to the testing environment
-      gmtDate.t.z // relative to the testing environment; relies on timeZone
-      gmtDate.t.Z // relative to the testing environment; relies on timeZone
+      gmtDate.t.timeZoneOffsetMM //should be("240") enter your timezone offset minutes here
```

Comment:
Why are these disabled?

Reply:
Those test cases are environment-dependent, so the user can edit them as they correspond to their respective environment. The issue is that determining the user's timezone, locale, etc. as a reference for these test cases would be redundant, because the only way we would have access to that is by using the very code we are testing.

Unless you know some third-party method of determining those factors, having the user "hard-code" their own data seems to be the best solution at this time.

Reply:
We should be setting the timezone for the tests, which should be possible on all platforms.

Reply:
Okay. I'll explore this further.